examples/chef/nrfconnect/prj.conf

```diff
@@ -46,9 +46,6 @@ CONFIG_RESET_ON_FATAL_ERROR=n
 # Disable Matter OTA DFU
 CONFIG_CHIP_OTA_REQUESTOR=n
 
-# Disable QSPI NOR
```

Comment:
Why is this change needed? QSPI NOR is only needed if you do the DFU.

Reply:
This change was blocking the build.